docker-compose.yml

```diff
@@ -6,3 +6,16 @@ services:
             context: "../bytepit-api"
         ports:
             - "8000:80"
+        depends_on:
+            -   database
+        environment:
+            - DB_CONNECTION_STRING=host=database dbname=rade_db user=postgres password=postgres
+
+    database:
+        image: postgres
+        environment:
+            POSTGRES_DB: rade_db
+            POSTGRES_USER: postgres
+            POSTGRES_PASSWORD: postgres
+        ports:
+            - "5432:5432"
```

Comment:
Put 5450 for our local port because some of us that have postgres server locally installed we have port 5432 already occupied. So e.g. "5450:5432"